transactional_licence_form/templates/start.html

```diff
@@ -1,5 +1,193 @@
 {% extends "layouts/base.html" %}
+{% load i18n %}
 {% block content %}
-  <h1>Placeholder page for the introductory text</h1>
-  <a href="{% url 'transactional-licence-form-steps' %}">Click to start</a>
+  <div class="standard-text-template container py-3">
+    <h1>Reusing case law data</h1>
+    <p>
+      <small>
+        Last updated on
+        <time datetime="10-04-2024">10th April 2024</time>
+      </small>
+    </p>
+    <p>
+      Open justice is a fundamental principle at the heart of the justice system - justice must not only be done, but it must also be seen to be done.
+      Access to judgments and decisions as data supports the principles of open justice by fostering transparency and promoting innovation in legal technology
+      and legal research. We have converted the judgments and decisions we hold into XML format, and made them freely available via the
+      <a href="https://github.com/nationalarchives/ds-find-caselaw-docs/blob/main/doc/openapi/public_api.yml">
+        Find Case Law API
+      </a>
+    </p>
+    <h3>You need a licence to reuse case law data</h3>
+    <p>
+      The National Archives, in collaboration with The Ministry of Justice and the Judicial Executive Board, have worked together to design a new licensing
+      framework for the reuse of case law as data. All licences are free but you may need to apply for a specific licence, depending on your reuse of the data.
+    </p>
+    <div class="info-box">
+      <h3 class="info-box__heading">
+        If you are not doing computational analysis, you maybe able to operate under the Open Justice Licence
+      </h3>
+      <div class="info-box__content">
+        <p>
+          Under this licence you can copy, publish, distribute and transmit case law data, and use the data commercially, for example, by combining it with other information,
+          or by including it in your own product or application, all with certain conditions.
+        </p>
+        <p>You can operate under the Open Justice Licence without needing to apply for one.</p>
+        <p>
+          <em>This licence does not cover computational analysis in bulk.</em>
+        </p>
+      </div>
+      <details class="govuk-details">
+        <summary class="govuk-details__summary">
+          <span class="govuk-details__summary-text">What is possible under the Open Justice Licence</span>
```

Comment:
This is a more general issue with the gds components that we have to look at, but this element is an example of it: the "focused" style is the standard GDS yellow highlight, rather than our own blue outline (annoyingly i can't screengrab a focused state to show you). We should add something to the gds overrides to use our own focus styles everywhere - I can help with this.

Reply:
oops sorry, another tiny thing here: the colour of the text on the dropdown component is slightly off: 
![image](https://github.com/nationalarchives/ds-caselaw-public-ui/assets/4279/ac2da2c4-72ce-4933-8251-26e8b57cbd0f)

Reply:
Hey Tim colour of text in dropdown header is blue as per the design by Bunnyfoot https://o88kd3.axshare.com/introduction_b.html

Reply:
Hey donna, yep, i think i'd treat the bunnyfoot designs more like wireframes for this type of thing: lets use the colour palette we have already!